src/utils/isAddressSigner/index.ts

```diff
@@ -4,8 +4,4 @@ import { isAddrEqual } from '../isAddrEqual'
 export const isAddressSigner = (
   address: Address,
   signers: Address[]
```

Comment:
We could potentially rename this whole method to `isAddressPresent(address: Address, addresses: Address[])` because it dosn't necessarily have to be used to find a signer